app/grandchallenge/challenges/templates/challenges/challenge_topbar.html

```diff
@@ -90,7 +90,7 @@
                     </div>
                     {% if "change_challenge" in challenge_perms %}
                         <div class="nav-item">
-                            <a class="nav-link {% if request.resolver_match.view_name == 'pages:update' or request.resolver_match.view_name == 'pages:delete' or request.resolver_match.view_name == 'pages:create' or request.resolver_match.view_name == 'update' or request.resolver_match.view_name == 'pages:list' or request.resolver_match.app_name == 'admins' or request.resolver_match.view_name == 'participants:list' or request.resolver_match.view_name == 'participants:registration-list' or request.resolver_match.view_name == 'evaluation:phase-update' or request.resolver_match.view_name == 'evaluation:create' or request.resolver_match.view_name == 'evaluation:method-list' or request.resolver_match.view_name == 'evaluation:method-create' or request.resolver_match.view_name == 'evaluation:evaluation-admin-list' or request.resolver_match.view_name == 'evaluation:method-detail' %}active{% endif %}"
+                            <a class="nav-link {% if request.resolver_match.view_name in 'pages:update,pages:delete,pages:create,update,pages:list,admins,participants:list,participants:registration-list,evaluation:phase-update,evaluation:create,evaluation:method-list,evaluation:method-create,evaluation:evaluation-admin-list,evaluation:method-detail,evaluation:combined-leaderboard-update,evaluation:combined-leaderboard-create' %}active{% endif %}"

```

Comment:
nice clean up, but you've missed the `request.resolver_match.app_name == 'admins'`, where `app_name`, not `view_name`, is matched.

Reply:
Good catch!